src/EssentialsPE/Loader.php

```diff
@@ -114,6 +113,7 @@ public function onEnable(): void{
             $this->getAPI()->fetchEssentialsPEUpdate(false);
         }
         $this->getAPI()->scheduleAutoAFKSetter();
+	$this->getLogger()->info("Plugin Enabled");
```

Comment:
seems unnecessary - also see indentation again

Reply:
Okay will remove it

Reply:
Removed